src/main/java/ch/njol/skript/classes/Changer.java

```diff
@@ -50,7 +51,7 @@ public boolean supportsKeyedChange() {
 	
 	abstract class ChangerUtils {
 
-		public static <T> void change(Changer<T> changer, Object[] what, Object @Nullable [] delta, ChangeMode mode) {
+		public static <T> void change(@NotNull Changer<T> changer, Object[] what, Object @Nullable [] delta, ChangeMode mode) {
```

Comment:
What is the contract for `what[]`?

Reply:
idk i just clicked the intellij button to add notnull
I presume it's a nullable array with notnull values though